graylog2-web-interface/src/components/search/MessageField.jsx

```diff
@@ -40,14 +32,15 @@ const MessageField = React.createClass({
 
     return (
       <span>
-        <dt key={`${key}Title`}>{key} {this._decorationMarker(key)}</dt>
```

Comment:
minor nitpick, I'd leave the decorated marker here instead of underneath the value.
with the new marker it's actually quite readable (I guess this change was made before the icon was changed into a text)